### PR TITLE
[Flow EVM] Adding FLOW token bridge event

### PR DESCRIPTION
**Pull request by ramtinms**
closes: #5537

### CODE REVIEW

**Review by sideninja — APPROVED** [review]
Nice!

File: fvm/evm/stdlib/contract.cdc [hunk 1]
```diff
@@ -7,6 +7,18 @@ contract EVM {
     access(all)
     event CadenceOwnedAccountCreated(addressBytes: [UInt8; 20])
 
+    /// FLOWTokenDeposit is emitted when FLOW tokens is bridged
```
Comment (janezpodhostnik) [inline fvm/evm/stdlib/contract.cdc:10]: I think we are using past tense for events. `FLOWTokensDeposited`.

File: fvm/evm/stdlib/contract.cdc [hunk 2]
```diff
@@ -224,6 +238,7 @@ contract EVM {
                 from: self.addressBytes,
                 amount: balance.attoflow
             ) as! @FlowToken.Vault
+            emit FLOWTokenWithdraw(addressBytes: self.addressBytes, amount: balance.inFLOW())
```
Comment (janezpodhostnik) [inline fvm/evm/stdlib/contract.cdc:241]: I would not emit an event if the balance is 0. I think I would prefer the whole function just exiting early if the balance is 0.

same with deposit.

File: fvm/evm/stdlib/contract.cdc (hunk 1 above)
Comment (ramtinms) [reply fvm/evm/stdlib/contract.cdc:10]: Update those to use past participle.

File: fvm/evm/stdlib/contract.cdc (hunk 2 above)
Comment (ramtinms) [reply fvm/evm/stdlib/contract.cdc]: added protection

**Review by janezpodhostnik — APPROVED** [review]

**Pull request merged**